src/common/googleCloud.ts

```diff
@@ -29,3 +31,104 @@ export const downloadJsonFile = async <T>({
 
   return JSON.parse(result);
 };
+
+export const userActiveStateQuery = `
+  with d as (
+    select uss.primary_user_id,
+    min(last_app_timestamp) as last_app_timestamp,
+    min(registration_timestamp) as registration_timestamp,
+
+      min(
+        case when period_end between date('2024-12-07' - interval 6*7 day) and '2024-12-07' then '1. active_last_6w'
```

Comment:
Why do we have all the hard coded dates here?
Also I kind of expected us to simply read from a export Dave would generate is that not the case?

Reply:
Modified this one, the static dates were mostly for testing. Applied params to the query to make the fetch dynamic 🙏

Reply:
So this query is the once-off one?
Or we always gonna full query all the time?

I sort of assumed we would just hit some results already?

Reply:
Yes, this is what we're going to use on a day-to-day basis. If I set the date today, we will get the correct values. Hence, we won't have this enabled until initial rollout.